src/tests/mocks/bors.rs

```diff
@@ -306,6 +306,136 @@ impl BorsTester {
         Ok(pr)
     }
 
+    pub async fn open_draft_pr(
```

Comment:
Could you please unify the implementation with `open_pr` above? They are almost the same and only differ in the true/false bool.